src/tests/test_utils/common_test_utils/src/common_utils.cpp

```diff
@@ -70,32 +72,34 @@ size_t getVmRSSInKB() {
 
 #else
 
-/// Parses number from provided string
-static int parseLine(std::string line) {
-    std::string res = "";
-    for (auto c : line)
-        if (isdigit(c))
-            res += c;
-    if (res.empty())
-        // If number wasn't found return -1
-        return -1;
-    return std::stoi(res);
-}
-
 size_t getSystemDataByName(char* name) {
+    auto parseLine = [](std::string line) -> size_t {
```

Comment:
Why do you need this code?
Maybe could you use `<<` operator from stream to get size_t?